src/pt/huntersscans/src/eu/kanade/tachiyomi/extension/pt/huntersscans/HuntersScans.kt

```diff
@@ -1,23 +1,247 @@
+                .flatMap { it }
+                .toSet()
+
+            chaptersLinks.map { chapterLink ->
+                SChapter.create().apply {
+                    name = chapterLink!!.toChapterName()
+                    setUrlWithoutDomain(chapterLink.toChapterAbsUrl())
+                }
+            }.toMutableList()
+        } catch (e: Exception) {
+            Log.e("HuntersScans", e.toString())
+            mutableListOf()
+        }
+    }
+
+    private fun containsDuplicate(chapters: List<SChapter>): Boolean {
+        return chapters.size != chapters.distinctBy { it.name }.size
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val chapters = chapterFromJScript(response)
+        val alwaysVisibleChapters = mutableSetOf<SChapter>()
+
+        val origin = response.request.url
+        var nextPage = 2
+
+        do {
+            if (chapters.size < 2) { break }
+
+            chapters += fetchChapterRequest(origin, nextPage).sortedBy { it.name.toInt() }
+
+            alwaysVisibleChapters += chapters.removeFirst()
+            alwaysVisibleChapters += chapters.removeLast()
+
+            nextPage++
+        }
+        while (!containsDuplicate(chapters) && chapters.isNotEmpty())
+
+        chapters += alwaysVisibleChapters
+
+        return chapters
+            .distinctBy { it.name }
+            .sortedBy { it.name.toInt() }
+            .reversed()
+    }
+
+    override fun chapterListSelector() = "script"
+
+    override fun imageUrlParse(document: Document) = ""
+
+    override fun latestUpdatesFromElement(element: Element) =
+        SManga.create().apply {
+            title = element.selectFirst("h3")!!.ownText()
+            thumbnail_url = element.selectFirst("img")?.absUrl("src")
+            setUrlWithoutDomain(element.selectFirst("a")!!.absUrl("href"))
+        }
+
+    override fun latestUpdatesNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun fetchLatestUpdates(page: Int): Observable<MangasPage> =
+        super.fetchLatestUpdates(page).map { it.removeNovels() }
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        val url = "$baseUrl/ultimas-atualizacoes".toHttpUrl().newBuilder()
+            .addQueryParameter("page", "$page")
+            .build()
+        return GET(url, headers)
+    }
+
+    override fun latestUpdatesSelector() = "main > div div:nth-child(2) > div.relative"
+
+    override fun mangaDetailsParse(document: Document) =
+        SManga.create().apply {
+            val container = document.selectFirst("div.container")!!
+            title = container.selectFirst("h2")!!.ownText()
+            thumbnail_url = container.selectFirst("img")?.absUrl("src")
+            genre = container.select("ul > li:nth-child(5) p").joinToString { it.ownText() }
+
+            val statusLabel = container.selectFirst("ul > li:nth-child(3) p")?.ownText()
+            status = statusManga.getOrDefault(statusLabel, SManga.UNKNOWN)
```

Comment:
```suggestion
            status = when (statusLabel) {
                "ongoing" -> SManga.ONGOING
                else -> SManga.UNKNOWN
            }
```
remove the statusManga map

Reply:
Resolved: https://github.com/keiyoushi/extensions-source/pull/1650/commits/2d6b2f344b8c1f5a11a3e2ee61346d277ecec643 and https://github.com/keiyoushi/extensions-source/pull/1650/commits/aaaf331e9ad9c49898b0bce485c535130ecebcac